djlsp/server.py

```diff
@@ -420,11 +422,15 @@ def initialized(ls: DjangoTemplateLanguageServer, params: InitializeParams):
 
 @server.feature(
     TEXT_DOCUMENT_COMPLETION,
-    CompletionOptions(trigger_characters=[" ", "|", "'", '"', "."]),
+    CompletionOptions(
+        trigger_characters=[" ", "|", "'", '"', "."], resolve_provider=True
+    ),
 )
 def completions(ls: DjangoTemplateLanguageServer, params: CompletionParams):
     logger.info(f"COMMAND: {TEXT_DOCUMENT_COMPLETION}")
     logger.debug(f"PARAMS: {params}")
+
+    TemplateParser.clear_completions_cache()
```

Comment:
Why the static method and not just do `_MOST_RECENT_COMPLETIONS.clear()` directly. Or keep the previous function that is called here.

Reply:
I have changed it back, youre right thats a more cleaner solution.